src/analysis/Environments/Environments.ts

```diff
@@ -221,6 +227,10 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
     await loadData();
   });
 
+  const pauseComputeAndRefresh: PauseButtonProps['pauseComputeAndRefresh'] = (compute: App | ListRuntimeItem) => {
+    void doPauseComputeAndRefresh(compute as DecoratedComputeResource);
+  };
```

Comment:
Why do we need this wrapper and cast? Could this be removed if the `compute` argument of `doPauseComputeAndRefresh` was typed `App | ListRuntimeItem`?

Reply:
The bigger issue is the return type... one is Promise based, the other is just void.